.pipelines/cni/datapath-test-templates/windows-datapath-template.yaml

```diff
@@ -0,0 +1,50 @@
+parameters:
+  clusterName: ""
+
+steps:
```

Comment:
is there actually anything windows specific about this or could we reuse this template across all scenarios?

Reply:
Specifically, lines 45 and 47 are currently the only windows specific lines. After looking into this I could add parameters to the load-test-template that calls it to ensure that this could be used for other k8s e2e.test testing. Thus, making it a better template.